### DISCUSSION TITLE
C#版本的SDK来了

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [peng-zhihui/ElectronBot#187] [Show and tell]
如题所示存c#版本的SDK初版来临，可以简单的发送指令，回头应该会再优化一下

# [代码仓库地址](https://github.com/GreenShadeZhang/ElectronBot.DotNet)

https://user-images.githubusercontent.com/24240675/170466406-069195c6-80b2-40be-a306-cdae20dbabeb.mp4

**Comment by carveybunt** [comment]
C 初学者，膜拜
